mail_tracking_mass_mailing/models/mail_mass_mailing_contact.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, fields
+
+
+class MailMassMailingContact(models.Model):
+    _inherit = 'mail.mass_mailing.contact'
+
+    tracking_email_ids = fields.Many2many(
+        string="Tracking emails", comodel_name="mail.tracking.email",
+        readonly=True)
+    email_score = fields.Float(
+        string="Email score",
+        compute="_compute_email_score", store=True, readonly=True)
+
+    @api.one
+    @api.depends('tracking_email_ids.state')
```

Comment:
Put also here just `'tracking_email_ids'` to assure it's recomputed when adding a new record.